eZ/Bundle/EzPublishCoreBundle/Resources/views/pagelayout.html.twig

```diff
@@ -7,7 +7,9 @@
     {% endif %}
     <title>{{ title|default( 'Home' ) }}</title>
     <meta name="generator" content="Ibexa DXP"/>
-    {% if content is defined and content.contentInfo.mainLocationId %}
+    {% if content is defined and location is defined and location is not null and location.id is not null %}
```

Comment:
is there really a possibility, that location.id is null?

Reply:
> Yes, that can be a case, hence [24a5b01](https://github.com/ezsystems/ezplatform-kernel/commit/24a5b01a732ac5e1da7615f44192938083e545ec)

We need more explanation before we can proceed with this. When it can be null?

Reply:
It is null by default when creating a new LP anywhere and kernel gives us a preview, this is a standard behavior even without this PR: 
![image](https://user-images.githubusercontent.com/22300504/216023615-f05926d4-9ac6-4d9a-968c-6207594cfb07.png)

This is a direct result of https://github.com/ezsystems/ezplatform-kernel/blob/1.3/eZ/Publish/Core/Helper/PreviewLocationProvider.php#L101-L109